front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/AboutReportWidget/messages.ts

```diff
@@ -7,7 +7,7 @@ export default defineMessages({
   },
   projectsLabel: {
     id: 'app.containers.admin.ReportBuilder.projectsLabel',
-    defaultMessage: 'Projects: {projectsList}',
+    defaultMessage: 'Project: {projectsList}',
```

Comment:
If you change a message, you also have to change its `id`- a crowdin quirk

Reply:
Even if it's not been translated yet? Np

Reply:
Yeah I think so... not 100% sure though. But I don't like to take risks with crowdin :)